### PR TITLE
docs: A one-liner about how Cube is packaged

### PR DESCRIPTION
**Pull request by igorlukanin**
We can add a link to "Architecture" later when we have it.

### CODE REVIEW

File: docs/content/Getting-Started/Overview.mdx [hunk 1]
```diff
@@ -7,6 +7,8 @@ menuOrder: 1
 isDisableFeedbackBlock: false
 ---
 
+Cube is a cloud-native application, designed to run as a set of containers.
```
Comment (hassankhan) [inline docs/content/Getting-Started/Overview.mdx:10]:
```suggestion
Cube is a cloud-native application, packaged and distributed as Docker images and designed to run in a containerized environment.
```

WDYT?

Comment (hassankhan) [reply docs/content/Getting-Started/Overview.mdx:10]: Could maybe add a link to https://docs.docker.com/get-started/overview/#images

Comment (igorlukanin) [reply docs/content/Getting-Started/Overview.mdx]: Sounds great!

File: docs/content/Getting-Started/Overview.mdx [hunk 2]
```diff
@@ -7,6 +7,8 @@ menuOrder: 1
 isDisableFeedbackBlock: false
 ---
 
+Cube is a cloud-native application, packaged and distributed as Docker images and designed to run in a containerized environment.
```
Comment (hassankhan) [inline docs/content/Getting-Started/Overview.mdx:10]: Could maybe also add a link to https://docs.docker.com/get-started/overview/#images?

Comment (igorlukanin) [reply docs/content/Getting-Started/Overview.mdx:10]: I suggest we omit this link here but maybe add it later to the Architecture page. Would be strange if we link to Docker's website first thing in Getting started.

Comment (hassankhan) [reply docs/content/Getting-Started/Overview.mdx:10]: Fair enough 😀

**Review by hassankhan — APPROVED** [review]

**Pull request merged**